lib/src/lightclient.rs

```diff
@@ -1028,11 +1029,8 @@ impl LightClient {
 
         // Add in all mempool txns
         tx_list.extend(wallet.mempool_txs.read().unwrap().iter().map( |(_, wtx)| {
-            use zcash_primitives::transaction::components::amount::DEFAULT_FEE;
-            use std::convert::TryInto;
-            
             let amount: u64 = wtx.outgoing_metadata.iter().map(|om| om.value).sum::<u64>();
-            let fee: u64 = DEFAULT_FEE.try_into().unwrap();
+            let fee = fee::get_default_fee(wallet.last_scanned_height());
```

Comment:
Is it possible for this to run before the last_scanned_height is known?

Should we be using last_scanned_height or the network_height? In other words, is the fee moreso a function of where the network is rather than where the wallet is?

Reply:
In this particular instance, it is a bit of a hack, because we're calculating it for mempool txns, which is where the wallet height is (since it will be synced).

Everywhere else, it is really the target block height, which is what the wallet's scanned height is, since the fee is used only when creating txns, which is possible only if the wallet is sycned.